gammapy/estimators/flux_map.py

```diff
@@ -136,6 +143,10 @@ def __init__(self, data, reference_model, meta=None, gti=None):
 
         self.meta = meta
         self.gti = gti
+        self.remove_failed = remove_failed
```

Comment:
Maybe choose a name that is closer to the `success` flag? Such as `filter_success_nan` or similar?